diff --git a/Cargo.toml b/Cargo.toml
index 8a26c6fc..3e6f43e9 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -139,7 +139,7 @@ vtimezones-rs = "0.3"
 ece = { version = "2.3", default-features = false, features = [
   "backend-openssl",
 ] }
-openssl = { version = "0.10", features = ["vendored"] }
+openssl = { version = "0.10" }
 async-std = { version = "1.13", features = ["attributes"] }
 similar-asserts = "2.0"
 insta = { version = "1.44", features = ["filters"] }
@@ -204,3 +204,9 @@ axum-extra.workspace = true
 headers.workspace = true
 http.workspace = true
 futures-util.workspace = true
+
+[profile.release]
+lto = 'thin'
+debug = false
+strip = true
+codegen-units = 1
